static/js/components/dashboard/FinancialAidCard.js

```diff
@@ -154,18 +155,20 @@ export default class FinancialAidCard extends React.Component {
     case FA_STATUS_PENDING_DOCS:
       return <div>
         <Grid>
-          <Cell col={12}>
-            Your cost is {price(coursePrice.price)} per course.
+          <Cell col={12} className="price-explanation">
+            <div>
+              Your cost is {price(coursePrice.price)} per course.
+            </div>
+            <a className="full-price" onClick={() => setConfirmSkipDialogVisibility(true)}>
+              Skip this and Pay Full Price
+            </a>
           </Cell>
         </Grid>
 
         <Grid className="financial-aid-box">
           <Cell col={12}>
-            Before you can pay, you need to verify your income by mailing or faxing one
-            of the following documents:
-            <ul>
-              <li>A notarized document verifying income</li>
-            </ul>
+            Before you can pay, you need to verify your income. Please mail or fax a
+            translated and notarized income tax or income statement document.
```

Comment:
How about this: "Before you can pay, you need to verify your income. Please mail or fax an English-translated and notarized income tax or income statement document."